proto/bpfman.proto

```diff
@@ -5,6 +5,7 @@ option go_package = "github.com/bpfman/clients/gobpfman/v1;v1";
 service Bpfman {
     rpc Load (LoadRequest) returns (LoadResponse);
     rpc Unload (UnloadRequest) returns (UnloadResponse);
+    rpc Attach (AttachRequest) returns (AttachResponse);
```

Comment:
I know we discussed this, and you said that Detach would come later.  Should you put it on the TODO list?

Reply:
it's TODO in a follow up PR so probably belongs in an issue after this one has landed.